### PR TITLE
404 page: 1st iteration(its creation)

### PR DESCRIPTION
**Pull request by Ladu-Lumori**

### CODE REVIEW

File: apps/web/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default function Home() {
+    return(
+    <div>
+    <p>"Sorry Mate!, Try some else : )"</p>
```
Comment (Emmanuel-Melon) [inline apps/web/pages/404.js:4]: Use Chakra UI components

**Review by Emmanuel-Melon — COMMENTED** [review]
Don't forget to attach a screenshot of the page. Also, try excluding this view from the layout.

File: apps/web/pages/404.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default function Home() {
```
Comment (Emmanuel-Melon) [inline apps/web/pages/404.js:1]: Rename page

File: apps/web/pages/404.js (hunk 1 above)
Comment (Ladu-Lumori) [reply apps/web/pages/404.js]: done that boss man

File: apps/web/pages/404.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Image, Img, Stack, Text, Button } from '@chakra-ui/react'
 
-export default function Home() {
+export default function ErrorPage() {
```
Comment (Emmanuel-Melon) [inline apps/web/pages/404.js:3]: ErrorPage is too generic.

**Review by Emmanuel-Melon — APPROVED** [review]
LGTM

**Comment by Ladu-Lumori** [comment]
404 image 1
how we are currently looking by 5th commit

**Pull request merged**